src/legacy/core_plugins/data/public/search/aggs/buckets/_terms_other_bucket_helper.test.ts

```diff
@@ -17,39 +17,73 @@
  * under the License.
  */
 
-import expect from '@kbn/expect';
-import ngMock from 'ng_mock';
 import {
   buildOtherBucketAgg,
   mergeOtherBucketAggResponse,
   updateMissingBucket,
-} from '../../buckets/_terms_other_bucket_helper';
-import { start as visualizationsStart } from '../../../../../../../core_plugins/visualizations/public/np_ready/public/legacy';
-import FixturesStubbedLogstashIndexPatternProvider from 'fixtures/stubbed_logstash_index_pattern';
+} from './_terms_other_bucket_helper';
+import { AggConfigOptions } from '../agg_config';
+import { AggConfigs } from '../agg_configs';
+import { BUCKET_TYPES } from './bucket_agg_types';
+import { IBucketAggConfig } from './_bucket_agg_type';
 
-const visConfigSingleTerm = {
-  type: 'pie',
+const indexPattern = {
+  id: '1234',
+  title: 'logstash-*',
+  fields: [
+    {
+      name: 'field',
+    },
+  ],
+} as any;
+
+const singleTerm = {
   aggs: [
     {
-      type: 'terms',
-      schema: 'segment',
-      params: { field: 'machine.os.raw', otherBucket: true, missingBucket: true },
+      id: '1',
+      type: BUCKET_TYPES.TERMS,
+      params: {
+        field: {
+          name: 'machine.os.raw',
+          indexPattern,
```

Comment:
why are indexPattern and filterable now passed in ? (whole field is an object, where before it was a string)

Reply:
I do not know how this worked before. But I think the field should be object. Also, I saw the object when debugging code.